opencensus/proto/agent/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenCensus Agent Proto
+
+This package describes the OpenCensus Agent protocol.
+
+## Architecture Overview
```

Comment:
Please add a TODO to avoid duplicating the same content between this readme and the readme in the opencensus-service. I think the overview belongs more to that repo, and here we should just refer to that. Happy to accept the PR as is and later move the text, up to you.

Reply:
Make sense, I left a TODO to move this section to the other README.